### PR TITLE
Release ironwood

### PR DESCRIPTION
**Pull request by slandrault**
## Purpose

This PR intends to update the edxapp to the first release of ironwood (2-oee).
It also update a script to not use a dump of the base's schema. This was done before to save time as migrate were too long. Migrations have been squashed in ironwood so it should be faster now.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,16 @@ Versioning](http://semver.org/spec/v2.0.0.html).
...
+## [5.14.3] - 2020-09-17
+
+### Added
+
+- Add `enable_sql_dump_loading` to `main.yml
+
 ### Changed
 
 - Upgrade `ansible` to `2.9.13`
+- Upgrade `edxapp` version to `ironwood.2-1.0.1`
```
Comment (jmaupetit) [inline CHANGELOG.md:16]:
```suggestion
- Upgrade default `edxapp` version to `ironwood.2-1.0.1`
```

File: group_vars/all/main.yml [hunk 2]
```diff
@@ -46,6 +46,12 @@ trashable_env_types:
   - "development"
   - "feature"
 
+# To be use with edxapp_sql_dump_url in edxapp config
+# This is useful when migration become too long to execute. We can
+# provide a sql dump of the schema to then load it instead of waiting
+# for migrate to be done (works only with trashable env)
+enable_sql_dump_loading: false
```
Comment (jmaupetit) [inline group_vars/all/main.yml]: If it defaults to `false` you are introducing a BC.

File: group_vars/all/main.yml [hunk 3]
```diff
@@ -46,6 +46,12 @@ trashable_env_types:
   - "development"
   - "feature"
 
+# To be use with edxapp_sql_dump_url in edxapp config
+# This is useful when migration become too long to execute. We can
```
Comment (jmaupetit) [inline group_vars/all/main.yml]:
```suggestion
# This is useful when migrations become too long to execute. We can
```

File: group_vars/all/main.yml [hunk 4]
```diff
@@ -46,6 +46,12 @@ trashable_env_types:
...
+# provide a sql dump of the schema to then load it instead of waiting
+# for migrate to be done (works only with trashable env)
```
Comment (jmaupetit) [inline group_vars/all/main.yml]:
```suggestion
# for the migrate command to end (works only for trashable environments)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,9 +8,16 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+## [5.14.3] - 2020-09-17
```
Comment (jmaupetit) [inline CHANGELOG.md]: This is not a fix, so you need to release a minor version:

```suggestion
## [5.15.0] - 2020-09-17
```

**Review by jmaupetit — CHANGES_REQUESTED** [review]

File: group_vars/all/main.yml (hunk 2 above)
Comment (slandrault) [reply group_vars/all/main.yml]: Right good point

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
```
Comment (lunika) [inline CHANGELOG.md:9]: In your commit message:

`s/To be use with/To be used with/`

`s/when migration become too long/when the migration becomes too long/`

`s/We can provide a sql dump of the schema to then load it/We can provide a sql dump of the schema and then load it`

File: group_vars/all/main.yml [hunk 7]
```diff
@@ -46,6 +46,12 @@ trashable_env_types:
   - "development"
   - "feature"
 
+# To be use with edxapp_sql_dump_url in edxapp config
```
Comment (lunika) [inline group_vars/all/main.yml]:
```suggestion
# To be used with edxapp_sql_dump_url in edxapp config
```

File: group_vars/all/main.yml [hunk 8]
```diff
@@ -46,6 +46,12 @@ trashable_env_types:
...
+# This is useful when migrations become too long to execute. We can
+# provide a sql dump of the schema to then load it instead of waiting
```
Comment (lunika) [inline group_vars/all/main.yml]:
```suggestion
# provide a sql dump of the schema and then load it instead of waiting
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -8,14 +8,16 @@ Versioning](http://semver.org/spec/v2.0.0.html).
...
```
Comment (lunika) [inline CHANGELOG.md:9]: Your commit title does not respect our release commits, should be:

`🔖(minor) bump release to 5.15.0`

File: CHANGELOG.md (hunk 6 above)
Comment (lunika) [inline CHANGELOG.md:9]: In the commit title, the subject is not `main` but `edxapp`. It is only related to edxapp application.

File: group_vars/customer/eugene/development/secrets/kibana.vault.yml [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+30333032366231656539653639316364383062363030363065666665383037356130663734323937
+38316437636266373035366438326561663438333130623937373338353237373738393965663135
+63653162346337313864376130326266376230313065353937333163303437353761393662363066
+66636338356532653061313664373662353438376138393336393564303633626334653137613764
+63383461313633623230636333376339323431346637386638376362656263333361396564643032
+32323237633638666564356337643539313835323333623237343965323534343337306638323939
+32323364373936316138326564356663356466366434623231663663663964653431313638356464
+66653864303363366433613038393337386133373065333335613039383030653035343830373065
+38323164383964623136643635613430316239363766323232623533636139383562626430376166
+38623437396366643632
```
Comment (jmaupetit) [inline group_vars/customer/eugene/development/secrets/kibana.vault.yml]: Why did you add this vault to this commit? Sounds like it comes out of the blue :sweat_smile:

**Review by jmaupetit — CHANGES_REQUESTED** [review]
Almost good to go.

1. this is not a patch but a minor release (see unresolved comment)
2. In your commits:
  - 2358f50f515d19cedc285471fdba6ebd3d2e0993: in the message `s/next/current/`
  - 2fb4dc8003c03355459b24152cdfcef4d075b527:
    * the title emoji does not comply with a new release (see @lunika's comment)
    * the message content does not reflect changes from the CHANGELOG

File: CHANGELOG.md [hunk 11]
```diff
@@ -10,6 +10,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Update ci test for edxapp
```
Comment (jmaupetit) [inline CHANGELOG.md]: As it's CI-related, I am not sure it's worth to be mentioned in the CHANGELOG.

File: CHANGELOG.md [hunk 12]
```diff
@@ -8,6 +8,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+## [5.14.3] - 2020-09-17
```
Comment (jmaupetit) [inline CHANGELOG.md]: You should also fix the date :sweat_smile:

File: apps/edxapp/tray.yml [hunk 13]
```diff
@@ -1,3 +1,3 @@
 metadata:
   name: edxapp
-  version: hawthorn.1-3.2.0
+  version: ironwood.2-1.0.1
```
Comment (jmaupetit) [inline apps/edxapp/tray.yml:3]: You commit title should be updated accordingly.

**Review by jmaupetit — APPROVED** [review]
